lr_face/evaluators.py

```diff
@@ -25,6 +25,29 @@ def plot_lr_distributions(predicted_log_lrs, y, savefig=None, show=None):
         plt.show()
 
 
+def plot_performance_as_function_of_resolution(scores, test_pairs, y_test,
+                                               savefig=None, show=None):
+    """
+    plots the scores as a function of the minimum resolution found on the
+    two images of the pair, coloured by ground truth
+    """
+    plt.figure(figsize=(10, 10), dpi=100)
+    min_resolutions = [min(min(pair.first.get_image().shape[:2]),
+                       min(pair.second.get_image().shape[:2])) for pair in
+                       test_pairs]
+    diff_resolutions = [min(pair.first.get_image().shape[:2])/
+                       min(pair.second.get_image().shape[:2]) for pair in
+                       test_pairs]
+    colors = list(map(lambda x: 'blue' if x else 'red', y_test))
+    plt.scatter(diff_resolutions, scores, c=colors)
+    plt.xlabel('10log LR')
+    if savefig is not None:
+        plt.savefig(savefig)
+        plt.close()
+    if show or savefig is None:
+        plt.show()
+
+
 def plot_tippett(predicted_log_lrs, y, savefig=None, show=None):
```

Comment:
deze functietitel kan generieker nu denk ik, want er worden niet alleen Tippett-plots geplot hier.

Reply:
? in dit functie zit toch alleen tippett?